File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        
        result, i = '', 0
        while i<len(word1) and i<len(word2):
            result += word1[i]+word2[i]
            i += 1
        result += word1[i:]+word2[i:]
        return result    

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        
        string result = "";
        int i = 0;
        while (i < word1.length() && i < word2.length()) {
            result += word1[i];
            result += word2[i];
            i++;
        }
        result += word1.substr(i) + word2.substr(i);
        return result;
        
    }
};